sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.query.sql

```diff
@@ -30,11 +31,19 @@ WITH _current AS (
     -- rightmost bit in 'days_since_seen' represents whether the user sent a
     -- baseline ping in the submission_date and similarly, the rightmost bit in
     -- days_active_bits represents whether the user counts as active on that date.
+    -- days_desktop_active_bits represents the official definition of used for desktop active user
     CAST(TRUE AS INT64) AS days_seen_bits,
     CAST(TRUE AS INT64) & CAST(durations > 0  AS INT64) AS days_active_bits,
     udf.days_since_created_profile_as_28_bits(
       DATE_DIFF(submission_date, first_run_date, DAY)
     ) AS days_created_profile_bits,
+    {% if app_name == "firefox_desktop" %}
+    CAST(TRUE AS INT64) &
+    CAST(browser_engagement_uri_count > 0 AS INT64) &
+    CAST(browser_engagement_active_ticks > 0 AS INT64) AS days_desktop_active_bits,
+    {% else %}
+    CAST(NULL AS INT64) AS days_desktop_active_bits,
```

Comment:
I wonder if we should just remove this else part here? Not sure it makes sense to add `days_desktop_active_bits` which is specifically firefox_desktop only field to every single app. But this would also require additional conditional logic inside `_previous` and in the final statement when we combine _previous with _current in the last_seen  and the corresponding schema templates.

Reply:
@scholtzan what do you think?

Reply:
Either way works.
Are we planning to track activity in a similar manner for mobile? Having something more generic like `days_active_bits` could be defined across applications.

Depending on how gnarly the logic would get, having it only for desktop, if it should be called `days_desktop_active_bits`, would be my preference.

Reply:
I do not think we are planning to have something similar for mobile. 

I will make the change to apply `days_desktop_active_bits` only for desktop

Reply:
@scholtzan When you get a chance could you do a final review before I merge. Thanks!